components/infobox/commons/infobox_weapon.lua

```diff
@@ -80,7 +80,7 @@ function Weapon:createInfobox()
 			}
 		},
 		Cell{name = 'Game Appearance(s)', content = {args.games}},
-		Customizable{id = 'custom', children = {}},
+		Customizable{id = 'customcontent', children = {}},
```

Comment:
i am against that change
rather adjust the /Custom as it was supposed to be

Reply:
Agreed

Reply:
Also not suitable for such a PR

Reply:
Removed from PR